.evergreen/auth_aws/aws_tester.py

```diff
@@ -23,12 +22,25 @@
 
 # This varies based on hosting EC2 as the account id and role name can vary
 AWS_ACCOUNT_ARN = "arn:aws:sts::557821124784:assumed-role/evergreen_task_hosts_instance_role_production/*"
-
-
-with open(os.path.join(HERE, 'aws_e2e_setup.json')) as fid:
-   CONFIG = json.load(fid)
-
-
+_USE_AWS_SECRETS = False
+
+if "USE_AWS_SECRETS" not in os.environ:
```

Comment:
Maybe instead we try to load from env if there is no `aws_e2e_setup.json` file?

Reply:
I like that idea, but what happens in the edge case where there is an `aws_e2e_setup.json` file but we want to load from env anyway?